gnosis/cowswap
# Add a loading indicator to Profile when loading data

# Summary

Closes #1632 

This adds a loading indicator to Profile page labels using the shimmer effect. It also sets an interval to fetch profile data each 5 minutes.

  # To Test

1. Connect wallet and open /profile page
2. Labels should glow while profile is loading
3. Wait 5 minutes to validate profile data is fetched again
